hack/import-restrictions.yaml

```diff
@@ -126,3 +126,24 @@
   - k8s.io/cli-runtime
   - k8s.io/client-go
   - k8s.io/sample-cli-plugin
+
+- baseImportPath: "./vendor/k8s.io/kube-controller-manager/"
+  allowedImports:
+  - k8s.io/apimachinery
+  - k8s.io/apiserver
+  - k8s.io/utils
```

Comment:
api, client-go

I believe utils is not need because it is no staging repo. Have you checked that?